gslib/commands/acl.py

```diff
@@ -339,6 +342,45 @@ class AclCommand(Command):
           'ch': _ch_help_text
       },
   )
+  CannedGcsAcl = ['private', 'public-read', 'project-private',
```

Comment:
Why make a static variable on the class?

Reply:
Use the immutable version of `set` please. Also, look at how constants are normally named.

Reply:
Looks like you switched to a set, but didn't make it immutable with `frozenset`